src/Linux/local_cache.cpp

```diff
@@ -268,6 +273,18 @@ static std::string get_file_name(const std::string& id)
     return g_cache_dirname + "/" + sha256(id);
 }
 
+std::string get_cached_file_location(const std::string& id){
+	return get_file_name(id);
+}
+
+std::string get_last_cache_read_expiry(){

```

Comment:
Do we still need this for production? This function's signature makes me think that it is going to return the last read expiry while it is returning a log message string. Might be better to return expiry value and then use it when constructing the log message.